package/post-install.sh

```diff
@@ -153,6 +153,23 @@ function verify_installation() {
         fail_print_and_exit
 }
 
+function verify_required_ext() {
+    # This comment has been added because without it the number of bytes of this script is 7432
+    # and the apk module of fpm generates a broken tar.
+    # For some reasons we have to reach at least a number of bytes greater or equal 7681 so I need
```

Comment:
This is the strangest comment I have ever seen 😆 

Can we include a link to some information about this quirk ?

Reply:
You are right, very unprofessional comment 😄 .

I moved it [here](https://github.com/DataDog/dd-trace-php/pull/1410/files#diff-78f4eefba9614400cb523865f64db2dc0dd3b1dc66ed89e6ef73d2ba2b937a8bR3-R12) and provided a more detailed investigation report, including the issue I opened on the fpm project https://github.com/jordansissel/fpm/issues/1866